### ISSUE TITLE
CSS extraction between brackets

### ISSUE THREAD

**Issue by tholman** [emmetio/extract-abbreviation#1]
Extracting the abbreviation of something like` body{bg|}` where `|` is the cursor results in `body{bg}` as the extraction. Should the correct behavior return just `bg`?

**Comment by ramya-rao-a** [comment]
I believe this module only works for html abbreviations and not css

@sergeche Do you plan to include support for css as well?

**Comment by sergeche** [comment]
Yes, current implementation works with markup abbreviations only but should also support CSS as well. Will implement CSS option later

**Comment by sergeche** [comment]
Just published update (v0.1.5) that supports stylesheet abbreviations (without `{}` and `[]`). New function interface: `extract(line, pos, { lookAhead, syntax })`, backward-compatible with `extract(line, pos, lookAhead)`. Syntax is either `markup` or `stylesheet`.

To extract stylesheet abbreviation: `extract('foo{bar}', 7, { syntax: 'stylesheet' })` → `bar`

**Comment by ramya-rao-a** [comment]
Thanks @sergeche 
Will test it out

**Comment by ramya-rao-a** [comment]
@sergeche The fix made for #2 is not getting applied in the latest module when using the old interface. This is because the `syntax` doesn't get set. I created a PR to fix this #4. Please do take a look
